runtimes/core/src/lib.rs

```diff
@@ -440,6 +451,18 @@ impl Display for ParseError {
 
 impl std::error::Error for ParseError {}
 
+fn infra_config_from_env() -> Result<Option<runtimepb::RuntimeConfig>, ParseError> {
+    let cfg_path = match std::env::var("ENCORE_INFRA_CONFIG_PATH") {
+        Ok(cfg) => cfg,
+        Err(_) => return Ok(None),
```

Comment:
How do you feel about:
```suggestion
        Err(std::env::VarError::NotPresent) => return Ok(None),
        Err(e) => return Err(ParseError::EnvVar(e)),
```